### PR TITLE
Clarify JSON/CLI instructions

### PR DESCRIPTION
**Pull request by rmoff**

### CODE REVIEW

File: docs/shared/connect.md [hunk 1]
```diff
@@ -2,4 +2,4 @@ Confluent Cloud offers pre-built, fully managed connectors that make it easy to
 This recipe shows some example data sources, but you can substitute your own connectors to connect to any supported data source.
 The principles are the same, just modify the connector configuration shown below to fit your situation (see [documentation](https://docs.confluent.io/cloud/current/connectors/index.html)).
 
-To run a fully managed connector to source data into Kafka, use the Confluent Cloud Console or [Confluent CLI](https://docs.confluent.io/confluent-cli/current/overview.html) command `confluent connect create --config <file>`. Each connector must be created separately.
+To run a fully managed connector to source data into Kafka, use the Confluent Cloud Console or [Confluent CLI](https://docs.confluent.io/confluent-cli/current/overview.html). Each connector must be created separately. If using the CLI put each JSON object below in its own file and then call each file with the command `confluent connect create --config <file>`.
```
Comment (ybyzek) [inline docs/shared/connect.md:5]: The less specific the text is, the more accurate it is ;)  Given that the next phase is ksqlDB-connect integration, I think it makes sense to go for less specific here.

```suggestion
To source data into Kafka, use the Confluent Cloud Console or [Confluent CLI](https://docs.confluent.io/confluent-cli/current/overview.html) to run each of the fully managed connectors shown below.
```

Comment (rmoff) [reply docs/shared/connect.md:5]: I respectfully disagree. If we're publishing this now then it should be as helpful as possible for someone using the platform now - we can make the change later as and when connect integration comes.

Comment (ybyzek) [reply docs/shared/connect.md:5]: The expectation is that users are running the recipes from the Console so there's also less incentive to show CLI at all.

Comment (rmoff) [reply docs/shared/connect.md:5]: Either we're helping people use the CLI, or we're not. I don't get why we'd show CLI commands but then not make the narrative helpful to that end.

But - I'll defer to you here, so feel free to close the PR if you want :)

Comment (ybyzek) [reply docs/shared/connect.md:5]: I'll close and reword in another PR

**Pull request closed without merging**